SimCalorimetry/EcalSimAlgos/interface/EcalTimeMapDigitizer.h

```diff
@@ -13,25 +13,32 @@
 #include "DataFormats/EcalDetId/interface/EcalSubdetector.h"
 #include "CalibFormats/CaloObjects/interface/CaloTSamplesBase.h"
 #include "DataFormats/EcalDigi/interface/EcalDigiCollections.h"
+#include "SimCalorimetry/EcalSimAlgos/interface/ComponentShapeCollection.h"
 
 class CaloSubdetectorGeometry;
 
 class EcalTimeMapDigitizer {
 public:
   struct time_average {
-    static const unsigned short time_average_capacity = 10;
+    static const unsigned short time_average_capacity = 10; // this corresponds to the number of BX -- double check this
+    static const unsigned short waveform_capacity = 10*25; // this will give a waveform with a granularity of 1ns
```

Comment:
Is this related to `WAVEFORMSAMPLES` above? If so consider defining a common constant.

Reply:
Yes, per comment above. How would I make a common constant for this and the constant in DataFormats?

Reply:
Can you use the one from the DataFormats instead?

Reply:
I changed the code around to use a constant from DataFormats.